src/components/Carousel.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const Carousel = props => {
+  return typeof props.images === 'object' && props.images.length ? (
+    <div>
```

Comment:
If we don't actually need a div here (which we should avoid) you can use [`React.Fragment`](https://reactjs.org/docs/fragments.html) (also [see shorthand](https://reactjs.org/docs/fragments.html#short-syntax) -- not sure if we can use it with our build system yet)